<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1265</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>80</y>
      <width>381</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="position">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Bookman Old Style</family>
         <pointsize>14</pointsize>
         <weight>3</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 25 14pt &quot;Bookman Old Style&quot;;</string>
       </property>
       <property name="text">
        <string>Współrzędne: 0 , 0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>800</width>
      <height>631</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>720</y>
      <width>450</width>
      <height>90</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item alignment="Qt::AlignVCenter">
      <widget class="QCommandLinkButton" name="pathfinder">
       <property name="text">
        <string>Szukaj</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignVCenter">
      <widget class="QCommandLinkButton" name="pause">
       <property name="text">
        <string>Pauza</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignVCenter">
      <widget class="QCommandLinkButton" name="refresh">
       <property name="text">
        <string>Odśwież</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>620</y>
      <width>391</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Wybierz wagę</string>
    </property>
    <widget class="QPushButton" name="force_0">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>0%</string>
     </property>
    </widget>
    <widget class="QPushButton" name="force_01">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>5%</string>
     </property>
    </widget>
    <widget class="QPushButton" name="force_02">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>10%</string>
     </property>
    </widget>
    <widget class="QPushButton" name="force_03">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>25%</string>
     </property>
    </widget>
    <widget class="QPushButton" name="force_04">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>60%</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>50</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(200, 248, 255)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="obstacle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>55</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Blokada</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>160</y>
      <width>381</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="percent_statement">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>262</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Wypełnij zadany procent siatki wybraną wagą</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="percent_open_nodes">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">qproperty-alignment: 'AlignVCenter | AlignRight';</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="percent_sign">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(200, 248, 255)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="statements">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>1191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 14pt &quot;Bookman Old Style&quot;;</string>
    </property>
    <property name="text">
     <string>Witaj! Wybierz punkt początkowy klikając LPM</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>290</y>
      <width>180</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="steps_label">
       <property name="text">
        <string>Kroki:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdSteps">
       <property name="styleSheet">
        <string notr="true">border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>290</y>
      <width>180</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="time_label">
       <property name="text">
        <string>Czas:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdTime">
       <property name="styleSheet">
        <string notr="true">border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>720</y>
      <width>331</width>
      <height>90</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="begin">
       <property name="styleSheet">
        <string notr="true">font: 25 10pt &quot;Bookman Old Style&quot;;</string>
       </property>
       <property name="text">
        <string>Początek:</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="end">
       <property name="styleSheet">
        <string notr="true">font: 25 10pt &quot;Bookman Old Style&quot;;</string>
       </property>
       <property name="text">
        <string>Koniec: </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QRadioButton" name="random">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>440</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Błądzenie losowe</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="random_force">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>470</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Błądzenie losowe z wagą</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="random_force_unique">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>500</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Błądzenie losowe z wagą - wariant drugi</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="percolates">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>410</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Perkolacja</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>810</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>© Amadeusz Oleszczak</string>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>statements</zorder>
   <zorder>groupBox_5</zorder>
   <zorder>groupBox_6</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_7</zorder>
   <zorder>random</zorder>
   <zorder>random_force</zorder>
   <zorder>random_force_unique</zorder>
   <zorder>percolates</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1265</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPercolation_memory">
    <property name="title">
     <string>Pamięć perkolacyjna</string>
    </property>
   </widget>
   <addaction name="menuPercolation_memory"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
